app/backend/src/couchers/servicers/communities.py

```diff
@@ -283,6 +285,7 @@ def ListPlaces(self, request, context, session):
         places = (
             node.official_cluster.owned_pages.where(Page.type == PageType.place)
             .where(Page.id >= next_page_id)
+            .where(Page.deleted is None)
```

Comment:
Compare to `None` with `deleted == None`. Or I think the actual right way to do this is to do `Page.deleted.is_(None)`, but `== None` works too. I believe `is None` might cause issues.

This is weird minutiae of SQLalchemy ORM mapping.